.github/workflows/verifast.yml

```diff
@@ -0,0 +1,42 @@
+name: VeriFast
```

Comment:
Could you add a section about VeriFast to our book? Please, see Kani example [here](https://model-checking.github.io/verify-rust-std/tools.html).

Reply:
I had a go at it. Feedback welcome!